src/cf-forms/src/atoms/label.less

```diff
@@ -3,10 +3,10 @@
 }
 
 .a-label__heading {
-    .heading-5();
+    .heading-4();
 
     display: block;
 
-    // Overwrites heading-5 margin.
-    margin-bottom: 10px;
+    // Overwrites heading-4 margin.
+    margin-bottom: unit(10px / @font-size, em);
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/519/commits/4cb65cb2a0a4b59cb70fce32bd08f90e6626ee38